robotiq_c_model_control/src/robotiq_c_model_control/params.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+"""
+Copyright 2018 by READY Robotics Corporation.
+All rights reserved. No person may copy, distribute, publicly display, create derivative works from or otherwise
+use or modify this software without first obtaining a license from the READY Robotics Corporation.
+"""
+import rospy
+from robotiq_c_model_control.constants import DEFAULT_MODBUS_IDS
+
+
+def get_supported_modbus_ids():
+    return rospy.get_param('/robotiq/gripper/supported_modbus_ids', DEFAULT_MODBUS_IDS)
```

Comment:
Should we make the param an alpha param? I don't mind either way

Reply:
I could be convinced either way. In some sense being able to target specific IDs isn't really an "alpha" feature -- it's more broadly useful outside of this specific instance and is something we can already support in the deployment_configuration.yaml file. It's only _slightly_ alpha in the sense that it's sort of intertwined with the bonnell stuff.

Reply:
Yeah, if we're going to add an alpha parameter, I'd like it to be whether multiple grippers are usable, as this _specific_ change seems independent of dual grippers.